.editorconfig

```diff
@@ -23,14 +22,10 @@ space_after_anonymous_functions = false
 spaces_in_brackets = false
 
 # No Standard.  Please document a standard if different from .js
-[**.yml, **.html, **.css]
+[{*.yml,*.html,*.css}]
 trim_trailing_whitespace = true
 indent_style = tab
 
 # No standard.  Please document a standard if different from .js
-[**.md]
-indent_style = tab
-
-# Standard at: 
-[Makefile]
+[*.md]
 indent_style = tab
```

Comment:
There is a newline.  If there wasn't it would indicate it with a little icon in the diff.